src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -582,6 +582,15 @@ namespace
         return mtl;
     }
 
+    bool is_node_animated(INode* node)
```

Comment:
Can the argument be made const?

Reply:
Ping :)

Reply:
Sorry @dictoon,
No, it can't be const. I tried but it doesn't allow because of the calls it makes inside the function.. Now I'm thinking if I should do the function const?

Reply:
Got it.

It's a free function, not a method, so it cannot be const.

Reply:
Yes, it doesn't work, `GetTMController()` doesn't allow parameter to be const.